src/parser.yy

```diff
@@ -109,6 +121,7 @@ void yyerror(bpftrace::Driver &driver, const char *s);
 %type <std::string> ident
 
 %right ASSIGN
+%right PLUSASSIGN
```

Comment:
Why does only this one need to be added here?

Reply:
Don't remember, it's been a few months since I made this change. I'll investigate.

Reply:
Guess it wasn't needed after all.